Modelica/Magnetic/FluxTubes/Examples/BasicExamples/QuadraticCoreAirgap.mo

```diff
@@ -0,0 +1,153 @@
+    annotation (Line(points={{-50,-10},{-40,-10}}, color={0,0,255}));
+  connect(rampCurrent.p, electricGround1.p)
+    annotation (Line(points={{-60,-10},{-50,-10}}, color={0,0,255}));
+  connect(rampCurrent.n, excitingCoil.p)
+    annotation (Line(points={{-60,10},{-40,10}}, color={0,0,255}));
+  connect(measuringCoil.n, electricGround2.p)
+    annotation (Line(points={{40,-10},{50,-10}}, color={0,0,255}));
+  connect(measuringCoil.p, voltageSensor.p)
+    annotation (Line(points={{40,10},{60,10}}, color={0,0,255}));
+  connect(electricGround2.p, voltageSensor.n)
+    annotation (Line(points={{50,-10},{60,-10}}, color={0,0,255}));
+  connect(excitingCoil.port_n, magFluxSensor.port_n)
+    annotation (Line(points={{-20,-10},{-20,-20}}, color={255,127,0}));
+  connect(magFluxSensor.port_p, magneticGround.port)
+    annotation (Line(points={{-20,-40},{-20,-50}}, color={255,127,0}));
+  annotation (Documentation(info="<html>
+<p>
+Educational example of a magnetic circuit containing an iron core and an airgap:
+</p>
+<p>
+<img src=\"modelica://Modelica/Resources/Images/Magnetic/FluxTubes/Examples/MagneticCircuit.png\" alt=\"Magnetic circuit with iron core and airgap\">
+</p>
+<p>
+A current ramp is applied in positive electric direction through the exciting coil, causing a rising magnetomotive force (mmf) in positive magnetic direction of the electromagnetic converter.  
+The mmf in turn causes a magnetic flux through the circuit in the direction indicated by the flux sensor. 
+From that magnetic flux, flux density can be calculated in every element of the magnetic circuit. Flux density is used to derive magnetic field strength.
+Magnetic field strength times length of the flux line gives magnetic potential difference of each element. 
+The sum of all magnetic potential differences is covered by the mmf of the exciting coil.
+</p>
+<p>
+Using the values shown in section Parameters, the results can be validated easily by analytic calculations:
+</p>
+<table>
+<tr><th>element   </th><th>cross section</th><th>length       </th><th>rel. permeability </th><th>B                   </th><th>H                                    </th><th>mmf              </th></tr>
+<tr><td>left leg  </td><td>a*a          </td><td>l - a        </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>upper yoke</td><td>a*a          </td><td>l - a        </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>right leg </td><td>a*a          </td><td>l - a - delta</td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>airgap    </td><td>a*a          </td><td>delta        </td><td>1                 </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
```

Comment:
```suggestion
<tr><td>airgap    </td><td>a*a          </td><td>delta        </td><td>1                 </td><td>flux / cross section</td><td>B/&mu;<sub>0</sub>                   </td><td>H*length         </td></tr>
```

Reply:
On my opinion this is otherwise a little misleading as it is not clear whether the table value `mu_r` or the Modelica parameter `mu_r` is meant here.